src/include/heapredirect.h

```diff
@@ -0,0 +1,274 @@
+/*
```

Comment:
Whole file should probably not be replaced?

Reply:
I'm quite confused as to how this happened, but `diff` says it's the same as the one in `vendor/Heap-Layers/wrappers` so I'm gonna feel safe deleting it